registry/build.sh

```diff
@@ -22,16 +22,13 @@ apk add --update-cache \
   xz-dev
 
 # install pip
-curl -sSL https://raw.githubusercontent.com/pypa/pip/7.0.3/contrib/get-pip.py | python -
-
-# workaround to python > 2.7.8 SSL issues
-pip install --disable-pip-version-check --no-cache-dir pyopenssl ndg-httpsclient pyasn1
+curl -sSL https://bootstrap.pypa.io/get-pip.py | python - pip==8.1.1
 
 # create a registry user
 adduser -D -s /bin/bash registry
 
 # add the docker registry source from github
-git clone -b new-repository-import-master --single-branch https://github.com/deis/docker-registry /docker-registry && \
+git clone -b deis-v1-lts --single-branch https://github.com/deis/docker-registry /docker-registry && \
```

Comment:
for other maintainer's reference, this contains commit https://github.com/deis/docker-registry/commit/c37b9ddd3bade4ce8420fcbcb74a370ceeefee16.